source/views/views.js

```diff
@@ -192,6 +192,15 @@ export const allViews: ViewType[] = [
 		tint: c.lavender,
 		gradient: c.seafoamToGrass,
 	},
+	{
+		type: 'view',
+		view: 'CovidView',
+		title: 'Covid Response',
+		icon: 'bug',
+		foreground: 'light',
+		tint: c.red,
+		gradient: c.orangeToRed,
+	},
```

Comment:
While this could live as a homescreen tile, I remember back to the work-in-progress feature on AAO for homescreen notices. I think it might make sense to have it live as a new homescreen tile in the small shape of a notice at the top of the view? This works too. I dunno.